### DISCUSSION TITLE
Planning release 0.18

### DISCUSSION THREAD

**Discussion by andsel** [moquette-io/moquette#885] [Show and tell]
Hi folks, I've created branch 0.18 to cut the the 0.18 release. Actually is drafted and contains mainly the MQTT 5 work done, plus Netty and H2 dependencies updates.
I've created a draft in https://github.com/moquette-io/moquette/releases, so please review and provide your feed back, I hope in a month to close the release and publish it.

cc @hylkevds

**Comment by hylkevds** [comment]
Sorry I didn't manage to test before the release, things were a bit chaotic before the holidays...

**Comment by andsel** [comment]
No worries, in the worst case we can do a patch release

Il ven 27 dic 2024, 19:59 Hylke van der Schaaf ***@***.***>
ha scritto:
